thoth/metrics_exporter/jobs/db.py

```diff
@@ -18,12 +18,14 @@
 """Knowledge graph metrics."""
 
 import logging
+from typing import Optional
 
 import thoth.metrics_exporter.metrics as metrics
 
 from .base import register_metric_job
 from .base import MetricsBase
 from ..configuration import Configuration
+from thoth.common import format_datetime
```

Comment:
please check format is the same between the query and the method in thoth-common

Reply:
`get_last_solver_datetime` returns a `datetime` object and `format_datetime` takes a `datetime` and returns a `str`, so the format should be good for setting the label